### PR TITLE
Update folium_vincent_markers.py

### PR DESCRIPTION
**Pull request by YasminFathy**
The old code has been commented to show the changes.

### CODE REVIEW

File: examples/folium_vincent_markers.py [hunk 1]
```diff
@@ -17,7 +17,9 @@
 # Binned wind speeds for NOAA 46050.
 bins = range(0, 13, 1)
 cuts = pd.cut(NOAA_46050['wind_speed_cwind (m/s)'], bins)
-ws_binned = pd.value_counts(cuts).reindex(cuts.levels)
+#ws_binned = pd.value_counts(cuts).reindex(cuts.values.levels) # cuts is a series, so values should be accessed
+# and also levels have been depreciated
```
Comment (ocefpaf) [inline examples/folium_vincent_markers.py]: No need to comment it out. You may just remove and we can rely on git's history to check the changes.

File: examples/folium_vincent_markers.py [hunk 2]
```diff
@@ -37,12 +39,23 @@
...
+# buoy_map.polygon_marker(location=[44.639, -124.5339], fill_color='#43d9de',
+#                         radius=12, popup=(vis2, 'vis2.json'))
+# buoy_map.polygon_marker(location=[46.216, -124.1280], fill_color='#43d9de',
+#                         radius=12, popup=(vis3, 'vis3.json'))
+# buoy_map.save(outfile='NOAA_buoys.html')
+
+# The example of Vincent/Vega Markers should be updated
+# https://github.com/python-visualization/folium
+buoy_map = folium.Map(location=[46.3014, -123.7390], zoom_start=7,tiles='Stamen Terrain')
+popup1 = folium.Popup(max_width=800,).add_child(folium.Vega(vis1, width=500, height=250))
```
Comment (ocefpaf) [inline examples/folium_vincent_markers.py]: :+1:

**Comment by ocefpaf** [comment]
@YasminFathy do you want to try to convert this example to a Jupyter notebook?

**Comment by YasminFathy** [comment]
Thanks @ocefpaf 
I have not try Jupyter notebook before, but I will try it within a couple of days and will convert this example.

**Comment by ocefpaf** [comment]
Thanks @YasminFathy for the contribution! I am merging this as-is and I will fix the linter errors in another PR. Feel free to try the Jupyter conversion any time :wink:

**Pull request merged**